Add Bright Cloud Studio, logo and package details

bright-cloud-studio opened the pull request:
Website studio from Western Massachusetts. Wondered how to get images working for our packages and finally figured it out.

xchs commented:
You might want to optimize the SVG logo a little bit with [SVGMO](https://jakearchibald.github.io/svgomg/)?

bright-cloud-studio commented:
I've never heard of [SVGMO](https://jakearchibald.github.io/svgomg/), essentially TinyPNG for SVG. Glad to know about it now, added it to the regular toolbox as it will come in handy many times in the future. I've updated logo.svg and the en.yml file.

bright-cloud-studio commented:
Just want the logo to show, going to delete these other files
